test/test_chatterjee_correlation.cpp

```diff
@@ -97,6 +98,33 @@ void test_threaded(ExecutionPolicy&& exec)
 
 #endif // BOOST_MATH_EXEC_COMPATIBLE
 
+template <typename Real>
+void test_paper()
+{
+    std::vector<Real> x = boost::math::generate_random_vector<Real>(1024, 0);
```

Comment:
On thing about the paper that might get us closer to the published value: The xs were spaced uniformly, not randomly.